src/uvw/tcp.hpp

```diff
@@ -220,7 +218,7 @@ class TcpHandle final: public StreamHandle<TcpHandle, uv_tcp_t> {
     }
 
 private:
-    enum { DEFAULT, FLAGS } tag;
+    enum { DEFAULT, FLAGS } tag{DEFAULT};
     unsigned int flags;
```

Comment:
I would update also this as `unsigned int flags{};` to be explicit about the fact that all the data members are initialized somehow.